irepela/homework_8/three_chefs.py

```diff
@@ -0,0 +1,53 @@
+# Details:  https://py.checkio.org/mission/3-chefs/share/33d5c11b5ff002a1645a6ee3b93ce00a/
+
+
+class AbstractCook:
+
+    def __init__(self):
+        self.total_food = 0
+        self.total_drink = 0
+        self.labels = {
+            "food": "Food",
+            "drink": "Drink"
+        }
```

Comment:
if params are for read only purpose or immutable, you can set them on class level